generate.py

```diff
@@ -54,23 +54,41 @@ def print_about_page(ydk_root, py_api_doc_gen, release, is_bundle):
     repo = Repo(ydk_root)
     url = repo.remote().url.split('://')[-1].split('.git')[0]
     commit_id = str(repo.head.commit)
+
+    if language == 'python':
+        cblock_language = 'sh'
```

Comment:
Can `cblock_language` be changed to `code_block_language_tag` so that it is more clear?